lib/sql_footprint/sql_statements.rb

```diff
@@ -3,12 +3,17 @@
 
 module SqlFootprint
   class SqlStatements
+    NEWLINE = "\n".freeze
     def initialize
       @statements = Set.new
     end
 
     delegate :to_a, :add, :sort, :count, to: :statements
 
+    def to_s
+      statements.sort.join(NEWLINE) + NEWLINE
```

Comment:
Well, we're not really doing anything w/ the file handle.  So I was thinking it would just serialize to string.  

How about some of these?
```
SqlStatementsSerializer.new(statements).to_s
SqlStatementsSerializer.new(statements).to_footprint
SqlStatementsSerializer.new(statements).to_footprint_s
```